src/marine/rads_adt2ioda.py

```diff
@@ -20,18 +20,36 @@
     IODA_CONV_PATH = Path(__file__).parent/'..'/'lib-python'
 sys.path.append(str(IODA_CONV_PATH.resolve()))
 
-import ioda_conv_ncio as iconv
+import ioda_conv_engines as iconv
 from orddicts import DefaultOrderedDict
 
+vName = "obs_absolute_dynamic_topography"
+
+locationKeyList = [
+    ("latitude", "float"),
+    ("longitude", "float"),
+    ("datetime", "string")
+]
+
+GlobalAttrs = {
+}
+
+VarDims = {
+    'adt': ['nlocs'],
```

Comment:
This is intended to be the `vName` I think, `adt` is not used anywhere is it? However, it is not coded correctly at the moment and thus is ignored. But will be fixed in my GSI-ncdiag PR.

Reply:
@CoryMartin-NOAA this has now been taken care of.